src/gui/executionlogwidget.cpp

```diff
@@ -32,75 +32,46 @@
 #include <QPalette>
 
 #include "base/global.h"
-#include "loglistwidget.h"
+#include "base/logger.h"
+#include "log/logfiltermodel.h"
+#include "log/loglistview.h"
+#include "log/logmodel.h"
 #include "ui_executionlogwidget.h"
 #include "uithememanager.h"
 
-ExecutionLogWidget::ExecutionLogWidget(QWidget *parent, const Log::MsgTypes &types)
+ExecutionLogWidget::ExecutionLogWidget(QWidget *parent, const Log::MsgTypes types)
```

Comment:
`Log::MsgTypes` is actually `QFlags<>`, I suspect it is small enough to be passed by value, I don't think it make sense that it will be larger than the underlying enum size.

Reply:
Ok, seems you're right.